authorization/authorization_integration_test.go

```diff
@@ -125,7 +125,7 @@ var _ = Describe("Authorized", func() {
 							resource1: PointTo(MatchFields(IgnoreExtras, Fields{
 								"Actions": MatchAllKeys(Keys{
 									action1: PointTo(MatchFields(IgnoreExtras, Fields{
-										"Allow": Equal(false),
+										"Allow": Equal(true),
```

Comment:
why did this need to change?

Reply:
change of data. The goal is to get the right answer, whatever it is